### PR TITLE
Configure resource requests and limits

### PR DESCRIPTION
**Pull request by vara2504**
https://tigera.atlassian.net/browse/EV-4687

Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by ctauchen** [comment]
@vara2504 You've partially reorganized the PR, but there's still a few things left.
* create a file at reference/component-resources/configure-resources.mdx
* Put the content of the include into configure-resources.mdx, with frontmatter, title, etc (what you had in the file you deleted)
* remove the include.

Once that's sorted, I'll get to the line edits. Thanks!

**Comment by vara2504** [comment]
@ctauchen My bad. I missed to add the untracked files. It should be good now.

File: calico-enterprise/reference/component-resources/configure-resources.mdx [hunk 1]
```diff
@@ -0,0 +1,1128 @@
...
+
+## Big picture
+
+Resource requests and limits are essential configurations for managing resource allocation and ensuring optimal performance of Kubernetes workloads. In {{prodname}}, these configurations can be customized using custom resources to meet specific requirements and optimize resource utilization.
+
+:::note
+It's important to note that the CPU and memory values used in the examples are for demonstration purposes and should be adjusted based on individual system requirements.
+:::
+
+
```
Comment (ctauchen) [inline calico-enterprise/reference/component-resources/configure-resources.mdx]: rm extra lines

File: calico-enterprise/reference/component-resources/configure-resources.mdx [hunk 2]
```diff
@@ -0,0 +1,1128 @@
...
+To configure resource specification for the [APIServerDeployment](../../reference/installation/api.mdx#operator.tigera.io/v1.APIServerDeployment), patch the installation CR using the below command:
+
+```bash
+$ kubectl patch apiserver tigera-secure  --type=merge --patch='{"spec": {"apiServerDeployment":{"spec": {"template": {"spec": {"containers":[{"name":"calico-apiserver","resources":{"limits":{"cpu":"1", "memory":"1000Mi"},"requests":{"cpu":"100m", "memory":"100Mi"}}},{"name":"tigera-queryserver","resources":{"limits":{"cpu":"1", "memory":"1000Mi"},"requests":{"cpu":"100m", "memory":"100Mi"}}}]}}}}}}'
+apiserver.operator.tigera.io/tigera-secure patched
+```
+This command sets the CPU request to 100 milliCPU (mCPU) and the memory request is set to 100 Mebibytes (MiB) while the CPU limit is set to 1 CPU and the memory limit is set to 1000 Mebibytes (MiB).
+
+Verifying Resources:
```
Comment (ctauchen) [inline calico-enterprise/reference/component-resources/configure-resources.mdx]:
```suggestion
#### Verification
```
Change globally.

File: calico-enterprise/reference/component-resources/configure-resources.mdx [hunk 3]
```diff
@@ -0,0 +1,1128 @@
...
+
+Resource requests and limits are essential configurations for managing resource allocation and ensuring optimal performance of Kubernetes workloads. In {{prodname}}, these configurations can be customized using custom resources to meet specific requirements and optimize resource utilization.
+
+:::note
+It's important to note that the CPU and memory values used in the examples are for demonstration purposes and should be adjusted based on individual system requirements.
+:::
+
+
+
+## APIServer
```
Comment (ctauchen) [inline calico-enterprise/reference/component-resources/configure-resources.mdx]:
```suggestion
## APIServer custom resource
```
Change globally for CRs. Helps visually to distinguish the major buckets from smaller types.

File: calico-enterprise/reference/component-resources/configure-resources.mdx [hunk 4]
```diff
@@ -0,0 +1,1128 @@
...
+    }
+  }
+}
+```
+
+## ApplicationLayer
+
+The [ApplicationLayer](../../reference/installation/api.mdx#operator.tigera.io/v1.ApplicationLayer) CR provides a way to configure resources for L7LogCollectorDaemonSet.
+
+Example Configurations:
```
Comment (ctauchen) [inline calico-enterprise/reference/component-resources/configure-resources.mdx]:
```suggestion
The [ApplicationLayer](../../reference/installation/api.mdx#operator.tigera.io/v1.ApplicationLayer) CR provides a way to configure resources for L7LogCollectorDaemonSet. The following sections provide example configurations for this CR.
```
Change globally.

File: calico-enterprise/reference/component-resources/configure-resources.mdx [hunk 5]
```diff
@@ -0,0 +1,1128 @@
...
+    },
+    "requests": {
+      "cpu": "100m",
+      "memory": "100Mi"
+    }
+  }
+}
+```
+
+### ############################
```
Comment (ctauchen) [inline calico-enterprise/reference/component-resources/configure-resources.mdx]: rm

**Review by ctauchen — COMMENTED** [review]
A few cosmetic changes. Otherwise LGTM!

**Comment by vara2504** [comment]
@ctauchen Addressed the review commands. Please take a look and let me know.

**Comment by ctauchen** [comment]
LGTM, thanks!

**Pull request merged**